src/fmu/dataio/_fmu_provider.py

```diff
@@ -200,6 +201,21 @@ def get_ert2_information(self):
             self.ert2["params"] = None
             logger.debug("parameters.txt was not found")
 
+        # Load restart_from information
+        if "RESTART_FROM_PATH" in environ:
```

Comment:
Can we test that this works as expected in some scenarios, e.g.
- `RESTART_FROM_PATH` is present and correct
- `RESTART_FROM_PATH` is present, but incorrect/anything/empty
- `RESTART_FROM_PATH` is not present

Reply:
I have updated the PR with the tests. Currently, it is setup to silently ignore invalid RESTART_FROM_PATH. But I am not sure if that's the behavior that we want. Any suggestions?